### PR TITLE
Add fixed packages in vulnerabilities details in packages endpoint.

### PR DESCRIPTION
**Pull request by TG1999**

### CODE REVIEW

File: vulnerabilities/api.py [hunk 1]
```diff
@@ -175,9 +203,9 @@ def filter_purl(self, queryset, name, value):
 
 
 class PackageViewSet(viewsets.ReadOnlyModelViewSet):
-    queryset = Package.objects.all()
+    queryset = Package.objects.all().order_by("id")
     serializer_class = PackageSerializer
-    paginate_by = 50
+    pagination.PageNumberPagination.page_size = 10
```
Comment (tdruez) [inline vulnerabilities/api.py:208]: This seems very wrong plus it's unrelated to this PR.

**Review by tdruez — CHANGES_REQUESTED** [review]
It looks like you have included changes completely unrelated to the goal of this PR:
- Pagination
- Ordering

File: vulnerabilities/api.py (hunk 1 above)
Comment (TG1999) [reply vulnerabilities/api.py:208]: I have cut a branch from #830, I will rebase this branch once #830 get's merged

File: vulnerabilities/api.py [hunk 2]
```diff
@@ -128,13 +117,52 @@ def to_representation(self, instance):
...
+            namespace=package.namespace,
+            type=package.type,
+            qualifiers=package.qualifiers,
+            subpath=package.subpath,
+            packagerelatedvulnerability__fix=True,
+        ).distinct()
+
+    def get_affected_vulnerabilities(self, package):
+        fixed_packages = self.get_fixed_packages(package=package)
+        qs = package.vulnerabilities.filter(packagerelatedvulnerability__fix=False)
```
Comment (tdruez) [inline vulnerabilities/api.py:137]: This is a bit confusing, `packagerelatedvulnerability__fix` is used with True in `get_fixed_packages` then overridden with False here?

Comment (TG1999) [reply vulnerabilities/api.py:137]: This qs is to get those vulnerabilities on the package that affects this that' why `packagerelatedvulnerability__fix` is set to False, whereas `get_fixed_packages` sends all the packages that matches name, namespace, type, qualifiers of that package and also fixes the vulnerability that's why it is set to True

Comment (tdruez) [reply vulnerabilities/api.py:137]: I get that, the confusing part is your approach of reusing the same base qs and overriding an existing lookup.

**Review by tdruez — APPROVED** [review]

**Pull request merged**